presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -341,6 +344,55 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+S3SelectPushdown
+^^^^^^^^^^^^^^^^
+S3SelectPushdown enables pushing down projection (SELECT) and predicate (WHERE) processing to `S3 Select <https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html>`_. With S3SelectPushdown
+Presto only retrieves the required data from S3 instead of entire S3 objects reducing both latency and network usage.
+
+Is S3 Select a Good Fit For My Workload?
+########################################
+Performance of S3SelectPushdown depends on the amount of data filtered by the query. Filtering a large number of rows should
+result in better performance. If the query doesn't filter any data then pushdown may not add any additional value and user will
+be charged for S3 Select requests. Thus, we recommend that you benchmark your workloads with and without S3 Select to see
+if using it may be suitable for your workload. By default, S3SelectPushdown is disabled and you should enable it in production
+after proper benchmarking and cost analysis. For more information on S3 Select request cost, please see `Amazon S3 Cloud Storage Pricing Doc <https://aws.amazon.com/s3/pricing/>`_.
+
+Use the following guidelines to determine if S3 Select is a good fit for your workload:
+
+* Your query filters out more than half of the original data set.
+* Your query filter predicates use columns that have a data type supported by Presto and S3 Select.
+  The ``TIMESTAMP``, ``REAL``, and ``DOUBLE`` data types are not supported by S3 Select Pushdown. We recommend using
+  the decimal data type for numerical data. For more information about supported data types for S3 Select,
+  see `Data Types <https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-data-types.html>` in the Amazon Simple Storage Service Developer Guide.
```

Comment:
The last part "in the Amazon Simple Storage Service Developer Guide" seems redundant since we are already linking to the documentation. How about changing it to

> see the Data Types documentation.

(where "Data Types" is the link as you already have)

Reply:
Sounds good. Refactored.